packages/react-router/package.json

```diff
@@ -41,6 +41,7 @@
     "history": "^4.7.2",
     "hoist-non-react-statics": "^2.5.0",
     "invariant": "^2.2.4",
+    "lodash.isequal": "^4.5.0",
```

Comment:
Can we use something else to achieve this? This [adds a fair number of bytes to the bundle](https://bundlephobia.com/result?p=lodash.isequal@4.5.0), so I'd like to avoid outside dependencies where we can.

Reply:
Sure -- I only used it to compare `match` [here](https://github.com/ReactTraining/react-router/pull/6341/files#diff-83865a255fa3e27147a9b92b078ee22cR109)

Do you have any other suggestions of how to compare `match`?